librz/core/cmd/cmd_debug.c

```diff
@@ -147,12 +144,9 @@ static const char *help_msg_dmi[] = {
 
 static const char *help_msg_do[] = {
 	"Usage:", "do", " # Debug (re)open commands",
-	"do", "", "Open process (reload, alias for 'oo')",
 	"dor", " [rz-run]", "Comma separated list of k=v rz-run profile options (e dbg.profile)",
 	"doe", "", "Show rz-run startup profile",
 	"doe!", "", "Edit rz-run startup profile with $EDITOR",
-	"doo", " [args]", "Reopen in debug mode with args (alias for 'ood')",
-	"doof", " [args]", "Reopen in debug mode from file (alias for 'oodf')",
 	"doc", "", "Close debug session",
 	NULL
 };
```

Comment:
In theory this can be removed completely if you removed the `do` code and moved it to rzshell.